types/evidence.go

```diff
@@ -29,6 +30,12 @@ type Evidence interface {
 	ValidateBasic() error     // basic consistency check
 }
 
+// From peppermint, will be removed eventually
+func RegisterEvidences(cdc *amino.Codec) {
```

Comment:
Please remove. No need (amino-related, protobufs don't need to be registered like this)

Reply:
Resolved